src/main/java/top/theillusivec4/curios/common/event/CuriosEventHandler.java

```diff
@@ -323,6 +331,44 @@ public void curioRightClick(PlayerInteractEvent.RightClickItem evt) {
       }
     });
   }
+  
+  @SubscribeEvent
+  public void looting(LootingLevelEvent event) {
+	  if (event.getDamageSource() != null)
+		  if (event.getDamageSource().getTrueSource() instanceof LivingEntity) {
+			  LivingEntity living = (LivingEntity) event.getDamageSource().getTrueSource();
+			  
+			  int totalLootingBonus = 0;
+			  
+			  if (CuriosApi.getCuriosHelper().getCuriosHandler(living).isPresent()) {
+					ICuriosItemHandler handler = CuriosApi.getCuriosHelper().getCuriosHandler(living).orElse(null);
+					Map<String, ICurioStacksHandler> curios = handler.getCurios();
+					
+					for (Map.Entry<String, ICurioStacksHandler> entry : curios.entrySet()) {
+						ICurioStacksHandler stacksHandler = entry.getValue();
+						String identifier = entry.getKey();
+						IDynamicStackHandler stackHandler = stacksHandler.getStacks();
+
+						for (int i = 0; i < stackHandler.getSlots(); i++) {
+							ItemStack stack = stackHandler.getStackInSlot(i);
+							LazyOptional<ICurio> curioCapability = CuriosApi.getCuriosHelper().getCurio(stack);
+							final int index = i;
+
+							if (!living.world.isRemote && !stack.isEmpty()) {
+								if (curioCapability.isPresent()) {
+									totalLootingBonus += curioCapability.orElseGet(null).getLootingBonus(identifier, living, stack, index);
+								} else {
+									totalLootingBonus += EnchantmentHelper.getEnchantmentLevel(Enchantments.LOOTING, stack);
```

Comment:
I'm not sure it's a great idea to default to increasing the looting bonus when this behavior is not explicitly defined. Modders may want the item to have the looting enchantment in general but not have it active in the slot. Of course, the opposite is valid as well. But I feel like this should default to no change if the capability is not present and let modders define the bonus in `getLootingBonus` when applicable. Otherwise, it's a hassle if a modder doesn't want this behavior and has to attach the capability just to disable it.